src/Main.scss

```diff
@@ -1,7 +1,21 @@
-$blue: #0000ff;
-$font-stack: Helvetica, sans-serif;
+$font-stack: Helvetica,
+sans-serif;
 
 body {
-  color: $blue;
   font: $font-stack;
 }
+
+.container {
```

Comment:
This class is unused. You can remove it.

Reply:
Unless you've used it somewhere I didn't see. In which case you can leave it.

Reply:
Good observation there. It was redundant.